src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,42 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class StringCalculator {
+
+    public static void main(String[] args) {
+        String stringNumber = "1111,2";
+        int sum = splitAndSum(stringNumber);
+        System.out.println(sum);
+    }
+
+    public static int splitAndSum(String stringNumber) {
+        if(stringNumber == null || stringNumber.isEmpty())  return 0;
+        int[] numbers = switchToNumbers(stringNumber);
+        checkValication(numbers);
+
+        return IntStream.of(numbers).sum();
+    }
+
+    public static int[] switchToNumbers(String stringNumber){
+        String customDelimiter = ",|:";
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(stringNumber);
+
+        if (m.find()) {
+            customDelimiter = m.group(1);
+            stringNumber = m.group(2);
+        }
+
+        int[] numbers = Stream.of(stringNumber.split(customDelimiter)).mapToInt(Integer::parseInt).toArray();
+
+        return numbers;
```

Comment:
```suggestion
        return Stream.of(stringNumber.split(customDelimiter))
                .mapToInt(Integer::parseInt)
                .toArray();
```

- Stream은 가독성을 위해 한줄에 한 행위씩 선언하는 것이 좋습니다
- 바로 return 해도 되는 구조네요 :)